draft-ietf-quic-transport.md

```diff
@@ -1741,7 +1741,8 @@ its Initial packet.  Including a token might allow the server to validate the
 client address without an additional round trip.  A client MUST NOT include a
 token that is not applicable to the server that it is connecting to, unless the
 client has the knowledge that the server that issued the token and the server
-the client is connecting to are jointly managing the tokens.
+the client is connecting to are jointly managing the tokens.  A client MAY use a
+token from any previous connection to that server.
```

Comment:
Maybe this sentence can be subsumed by the first sentence of the same paragraph that discusses the applicability of the token?